src/api/app/services/trigger_controller_service/token_extractor.rb

```diff
@@ -16,9 +16,15 @@ def call
 
     private
 
+    def extract_token_string_from_authorization_header
+      http_auth = @http_request.env['HTTP_AUTHORIZATION']
+
+      http_auth.slice(6..-1) if http_auth.present? && http_auth[0..4] == 'Token'
```

Comment:
Does it have to be strict by enforcing the use of `Token`?

Reply:
It seems to me that it was a change that was not planned. The current documentation doesn't reflect that change, either: https://build.opensuse.org/apidocs/index#128

The current alternative accepts any combination of five first characters. If we go for an option of using an authentication header, we should follow a `Authorization: <type> <credentials>` convention, as described [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication). It could colide with some other authentication scheme which a type of five characters.

Reply:
Alright, let's enforce what we have in the API documentation.